cloudify_rest_client/deployments.py

```diff
@@ -296,3 +314,21 @@ def set_visibility(self, deployment_id, visibility):
             '/deployments/{0}/set-visibility'.format(deployment_id),
             data=data
         )
+
+    def set_site(self, deployment_id, site_name=None, detach_site=False):
+        """
+        Updates the deployment's site
+
+        :param deployment_id: Deployment's id to update.
+        :param site_name: The site to update
+        :param detach_site: True for detaching the current site, making the
+                            deployment siteless
+        :return: The deployment.
+        """
+        data = {'detach_site': detach_site}
+        if site_name:
```

Comment:
What will happen if detach-site and site are equal?
maybe in that case a warning message will be printed and save the http call?